src/app/basket/basket.module.ts

```diff
@@ -3,15 +3,15 @@ import {CommonModule} from '@angular/common';
 import {RouterModule} from '@angular/router';
 import {BasketComponent} from './basket.component';
 import {ComplexNavigatorModule} from '../complex/complex-results/complex-navigator/complex-navigator.module';
-import {ProgressSpinnerModule} from '../shared/loading-indicators/progress-spinner/progress-spinner.module';
+import {ProgressSpinnerComponent} from '../shared/loading-indicators/progress-spinner/progress-spinner.component';
 
 @NgModule({
   imports: [RouterModule.forChild([
     {path: '', component: BasketComponent}
   ]),
     CommonModule,
     ComplexNavigatorModule,
-    ProgressSpinnerModule
+    ProgressSpinnerComponent
```

Comment:
Why the component and not the module? We are importing `CommonModule` and `ComplexNavigatorModule`. Why import `ProgressSpinnerComponent` instead of `ProgressSpinnerModule`?

Reply:
I see you deleted the module. Shouldn't it then go in declarations instead of imports?

Reply:
I made the ProgressSpinnerComponent a standalone component as that was essentially what it already was, just made it official let's say